src/library/Asset.ts

```diff
@@ -1,16 +1,19 @@
 import { AssetMethods } from "./AssetMethods";
 import Assets from "../assets.json";
 import { AssetGroupModel } from "../types/assets.t";
+import {AssetModel} from "../types/assets.t";
+
+
 
 export class Asset {
 
   private options;
-  private asset;
+  public asset;
```

Comment:
@byterose This was private before, but I'm not sure why it would be. The reason I changed it is because this property was being accessed in a test. Let me know if this needs to be changed back.

Reply:
I made it public in my pr, thank you for the hint!